critiquebrainz/frontend/templates/bb_edition_group/entity.html

```diff
@@ -60,6 +60,51 @@ <h4 style="margin-bottom:0;">{{ _('Reviews') }}</h4>
         {% endif %}
       </ul>
     {% endif %}
+
+    {% if works is defined and works %}
+      <h4>{{ _('Literary Works') }}</h4>
+      <table class="table table-condensed table-hover">
+        <thead>
+          <tr>
+            <th>{{ _('Name') }}</th>
+            <th>{{ _('Languages') }}</th>
+            <th>{{ _('Type') }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for work in works %}
```

Comment:
do works in an edition have an ordering? I don't see an `ORDER BY` clause in the python method.

Reply:
No, Works for Edition does not have an ordering.